scripts/fdi_local_install.sh

```diff
@@ -7,5 +7,7 @@ echo "Set site uuid..."
 ahoy drush config-set "system.site" uuid "$SITE_UUID" -y
 echo "Importing config..."
 if [ -f ./config/sync/core.extension.yml ] ; then ahoy drush cim -y ; fi
+echo "Translations..."
+ahoy drush locale-check && ahoy drush locale-update
```

Comment:
Didn't we decided to not add this to the install and run it manually when needed?